centralserver/i18n/management/commands/update_language_packs.py

```diff
@@ -1,858 +0,0 @@
-"""
```

Comment:
Could you open a PR on ansible-playbooks to stop running this, if that's all happening (for newer versions) on the build server?